test/parallel/test-repl-history-navigation.js

```diff
@@ -554,6 +556,27 @@ const tests = [
     expected: [],
     clean: false
   },
+  {
+    env: { NODE_REPL_HISTORY: defaultHistoryPath },
+    test: [
+      'const util = {}',
+      ENTER,
+      'proc',
+      RIGHT,
+      SIGINT
+    ],
```

Comment:
Don't use `SIGINT`, it could be a problem if we want to add more test cases in the future.
```suggestion
    test: ['const util = {}', ENTER,
           'proc', RIGHT, ENTER],
```